shared/@hedvig-ui/Popover/popover.tsx

```diff
@@ -62,6 +62,7 @@ export const Popover: React.FC<PopoverProps> = ({
   disable,
   onOpen,
   onClose,
+  style,
   children,
```

Comment:
```suggestion
  children,
  ...props
```

Something like this and just spread it to `Contents` instead of explicitly specifying `style`